libcudacxx/test/libcudacxx/cuda/ptx/generated/get_sreg.h

```diff
@@ -57,7 +57,7 @@ __global__ void test_get_sreg(void** fn_ptr)
 #endif // __cccl_ptx_isa >= 130
 
 #if __cccl_ptx_isa >= 200
-  NV_IF_TARGET(NV_PROVIDES_SM_35,
+  NV_IF_TARGET(NV_PROVIDES_SM_50,
                (
```

Comment:
Please revert the changes in this generated file, since we continue to support `NV_PROVIDES_SM_35` for now. I will also bring in some changes to these generated files shortly and want to reduce the chance of conflict. Thx!